### ISSUE TITLE
cooler name

### ISSUE THREAD

**Issue by torpak** [fluix-dev/sway-borders#25]
since you are mostly adding visually pleasing features to sway, how about you call the result "swag"?

**Comment by fluix-dev** [comment]
Not a bad idea actually. I'll consider it some more once I get around to the other visual features (because right now it _is_ just borders).

**Comment by qubidt** [comment]
I kinda like the callback to i3-gaps. Makes the relationship pretty clear if you're familiar with i3. swag ain't bad tho

**Comment by fluix-dev** [comment]
Yeah, `sway-borders` is named as such because of `i3-gaps`, but once more and more features come (if they do), I'm not sure how well it'll fit. On the other hand, once people make the connection that `sway-borders` == `sway+graphics stuffs` it's not a problem.

I also worry that `swag` might be confusing for people searching for issues or that it may be confused with an entirely separate compositor.

**Comment by al-caveman** [comment]
If the name is to change, I suggest to pick a Google-able name.  Both sway and swag are dictionary words.

**Comment by fluix-dev** [comment]
While both `sway` and `swag` are not searchable on their own, adding `wm` as a keyword brings them up. This seems to be true of multiple other window managers (ex. `mutter`, `i3`) as well. Somewhat pleasantly, both `i3-gaps` and `sway-borders` lead to the correct projects, so if search-ability is a concern (which I haven't quite made up my mind on yet, linux window managers are quite niche after all), the current name may actually be best.

**Comment by Hubro** [comment]
Another suggestion to retain the naming convention: `sway-bling`

**Comment by lovelymono** [comment]
Another idea could be `sway-cooler` like a "way cooler sway".
